samza-sql/src/main/java/org/apache/samza/sql/runner/SamzaSqlApplication.java

```diff
@@ -69,13 +66,13 @@ public void describe(StreamApplicationDescriptor appDescriptor) {
       // 3. Translate Calcite plan to Samza stream operators
       QueryTranslator queryTranslator = new QueryTranslator(appDescriptor, sqlConfig);
       SamzaSqlExecutionContext executionContext = new SamzaSqlExecutionContext(sqlConfig);
-      Map<String, SamzaRelConverter> converters = sqlConfig.getSamzaRelConverters();
+      int queryId = 0;
```

Comment:
Can you check with Shenoda to make sure we use the same queryId for diagnostics as well?